cpp/src/io/utilities/thread_pool.hpp

```diff
@@ -0,0 +1,401 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
```suggestion
 * Copyright (c) 2021, NVIDIA CORPORATION.
```

Reply:
It's already July? Where did this year go?